Android/app/src/main/java/io/github/project_travel_mate/MainActivity.java

```diff
@@ -108,8 +109,10 @@ protected void onCreate(Bundle savedInstanceState) {
         mToken = mSharedPreferences.getString(USER_TOKEN, null);
         mHandler = new Handler(Looper.getMainLooper());
 
-        DailyQuotesManager.checkDailyQuote(this);
-
+        boolean isNetworkConnected = Utils.isNetworkConnected(this);
```

Comment:
remove this variable `isNetworkConnected`
directly use `Utils.isNetworkConnected(this)` in if statement

Reply:
done i changed it